### ISSUE TITLE
On sandbox unable to align a competency with an learning opp

### ISSUE THREAD

**Issue by mparsons-ce** [cassproject/cass-editor#1024]
@woodkri 
In the sandbox, we are unable to align competencies to learning opps, etc. 
This feature works on both staging and production. 
![image](https://user-images.githubusercontent.com/58825866/150010873-dab0b514-a84b-4e63-a9c1-67a74457cee8.png)

**Comment by woodkri** [comment]
Yes, I noticed this on staging recently and have fixed it. The fix isn't up on sandbox yet though.
